Cargo.toml

```diff
@@ -201,10 +201,14 @@ members = [
     "data_model/samples/*"
 ]
 
+[profile.release]
+debug-assertions = true
+
 [profile.deploy]
 inherits = "release"
 strip = "symbols"
 lto = true
+debug-assertions = false
```

Comment:
My understanding is that now in `wasm/Cargo.toml`, `deploy` takes over `release` and `release` is overridden to have some debug features, but why does it affect the workspace `Cargo.toml` ?

Reply:
That was left from the debugging. Removed in a new commit.